### PR TITLE
Remove Python 2 references in exercise README template

### PR DESCRIPTION
**Pull request by tqa236**
@cmccandless I deleted all Python 2.7 references in README. Does this look good for you? I can do that for other files too if this works.

### CODE REVIEW

**Comment by cmccandless** [comment]
@tqa236 Which issue are you targeting? Your title suggests #1704, but your changes apply more to #1705.

**Comment by tqa236** [comment]
Hello, I think it's more about #1705 than #1704. Unfortunately I haven't modified the file `exercise_readme.go.tmpl` yet but I ran a script to remove all Python 2 related path in the README of each exercise. I didn't know that the READMEs are automatically generated. What do you suggest that I should do here?

**Comment by cmccandless** [comment]
I would suggest the following workflow:
1. Leave your current changes in place. Since it is all generated files, they won't cause issues if something else changes later 
1. Make the appropriate changes to `exercise_readme.go.tmpl` (we might exchange a few rounds of feedback on this one)
1. Once the template is settled, use `configlet generate` to apply the template and commit any changes to the index
1. I'll do a final once-over. If everything looks good, we'll merge!

**Comment by tqa236** [comment]
The workflow is fine for me. I just deleted the line about Python 2 as a starting point for discussion. There's some comparisons of Python 2 and 3 that in my opinion unnecessary too. I want to hear your opinion about that.

File: config/exercise_readme.go.tmpl [hunk 1]
```diff
@@ -23,7 +23,6 @@ raise Exception("Meaningful message indicating the source of the error")
 
 To run the tests, run the appropriate command below ([why they are different](https://github.com/pytest-dev/pytest/issues/1629#issue-161422224)):
 
-- Python 2.7: `py.test {{ .Spec.SnakeCaseName }}_test.py`
 - Python 3.4+: `pytest {{ .Spec.SnakeCaseName }}_test.py`
```
Comment (cmccandless) [inline config/exercise_readme.go.tmpl]: Track documentation will mention elsewhere that we support Python3.4+, I'm not sure it's necessary to do so here. Something like the following will suffice for this template:

> To run the tests, run `pytest {{ .Spec.SnakeCaseName }}_test.py`

File: config/exercise_readme.go.tmpl [hunk 2]
```diff
@@ -23,7 +23,6 @@ raise Exception("Meaningful message indicating the source of the error")
...
 
 Alternatively, you can tell Python to run the pytest module (allowing the same command to be used regardless of Python version):
```
Comment (cmccandless) [inline config/exercise_readme.go.tmpl]: Within the same mindset as the above comment, I don't think we need the part in parenthesis here:

> Alternatively, you can tell Python to run the pytest module:

**Comment by cmccandless** [comment]
Template looks good. Go ahead and generate the READMEs again.

**Comment by tqa236** [comment]
How can we do that exactly? I run the command `configlet generate` but it doesn't work.

**Comment by cmccandless** [comment]
You'll need to:
1. Clone exercism/problem-specifications
2. Run `configlet generate -p $WHERE_YOU_CLONED .` in the root directory for this repository

**Comment by tqa236** [comment]
I generate the new READMEs with the latest `problem-specifications`, so there's some unrelated change as well.

**Comment by tqa236** [comment]
Do you know why the CI doesn't mark as completed? It is already completed in the Travis CI website.

**Comment by cmccandless** [comment]
@tqa236 I'm not sure why it doesn't mark as completed, but it is supposed to block incomplete from merging, and it didn't block me, so the backend is working correctly.

Thanks for working on this!

**Pull request merged**